.eslintrc.js

```diff
@@ -40,5 +40,9 @@ module.exports = {
     'testing-library/await-async-query': 'error',
     'testing-library/no-await-sync-query': 'error',
     'testing-library/no-dom-import': 'off',
+    'linebreak-style': 0,
+  },
+  env: {
+    browser: true,
```

Comment:
What's the context here, is this for the library?

Reply:
eslint was complaining about windows being global here.

```bash
option === "issues" ? window.open(`${repoLink}/issues`) : window.open(repoLink);
    if (option === 'issues') return window.open(`${repoLink}/issues`);
    return window.open(repoLink);
  };
```
And setting the browser as true resolved it.

Reply:
That is an opaque solution. Perhaps we could do this instead in the package.json. 

```json
"eslintConfig": {
  "globals": {
    "window": true
  }
}
```

Reply:
That's nicer.

Reply:
put this in `.eslintrc.js` please!